xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -52,12 +52,12 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesConsumed">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The byte read-only span containing the compressed data.</param>
```

Comment:
Since the parameter type is already shown, does it need to be repeated here, i.e., could this be simply "The compressed data."? Same comment on the following parameter.

Reply:
I see what you mean, makes sense. Other similar methods, like [DeflateStream.BeginWrite](https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.deflatestream.beginwrite?view=netcore-3.0) or [GZipStream.Write](https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.gzipstream.write?view=netcore-3.0#System_IO_Compression_GZipStream_Write_System_Byte___System_Int32_System_Int32_), don't really mention the data type, only that it's a buffer.

How about:

```suggestion
        <param name="source">A buffer containing the compressed data.</param>
```